pinot-common/src/main/java/org/apache/pinot/common/config/provider/TableCache.java

```diff
@@ -363,14 +366,13 @@ private void putTableConfig(ZNRecord znRecord)
 
   private void putLogicalTableConfig(ZNRecord znRecord)
       throws IOException {
-    LogicalTableConfig logicalTableConfig = LogicalTableUtils.fromZNRecord(znRecord);
+    LogicalTableConfig logicalTableConfig = LogicalTableConfigUtils.fromZNRecord(znRecord);
     String logicalTableName = logicalTableConfig.getTableName();
+    _logicalTableConfigInfoMap.put(logicalTableName, new LogicalTableConfigInfo(logicalTableConfig));
```

Comment:
What is the significance of this change ? I assumed that standardizing on lowercase as a key would be better.

Reply:
The `_logicalTableConfigInfoMap` keep the logical table name as it is but `_logicalTableNameMap` store the lowercase to original table name mapping.

This is how is done for table config too.